ecs_fargate/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - ECS Fargate
 
+## 3.4.0 / 2023-06-15 / Agent 7.46.0
+
+* [Added] Add ephemeral storage metrics and change ECS metadata endpoint version from v2 to v4. See []().
```

Comment:
Missing the github number

Reply:
IIRC, changelogs in `integrations-core` repo get generated automatically after merge.